endpoints/getting-started-grpc/client/main.go

```diff
@@ -61,8 +68,33 @@ func main() {
 	defer conn.Close()
 	c := pb.NewGreeterClient(conn)
 
+	if *keyfile != "" {
+		log.Printf("Authenticating using Google service account key in %s", *keyfile)
+		sakey, err := ioutil.ReadFile(*keyfile)
+		if err != nil {
+			log.Fatalf("Unable to read service account key file %s: %s", *keyfile, err)
```

Comment:
`%v` for the second formatting directive (for err)

Reply:
Done